src/AddTodoForm.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+export default function AddTodoForm(props) {
+  const handleAddTodo = (event) => {
+    event.preventDefault();
+    var todoTitle = event.target.elements.title.value;
+    console.log(todoTitle);
+    console.log(event);
```

Comment:
you should consider removing the log statements here before merging your code :)

Reply:
Oops, disregard this. I see that you're supposed to keep the logs in here for the assignment. My bad